parts/common/components.json

```diff
@@ -470,6 +470,10 @@
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/azure-cloud-node-manager",
           "latestVersion": "v1.31.2",
           "previousLatestVersion": "v1.31.1"
+        },
```

Comment:
Since renovate can only bump versions and cannot create new release, we don't have to rely on renovate to introduce this cache.

Reply:
You are right